content/docs/hooks-custom.md

```diff
@@ -184,24 +183,24 @@ function ChatRecipientPicker() {
 
-Because the `useState` Hook call gives us the latest value of the `recipientID` state variable, we can pass it to our custom `useFriendStatus` Hook as an argument:
+Como la llamada al Hook `useState` nos da el ultimo valor de la variable de estado `recipientID`, podemos pasarla a nuestro Hook personalizado `useFriendStatus` como un argumento:
 
 ```js
   const [recipientID, setRecipientID] = useState(1);
   const isRecipientOnline = useFriendStatus(recipientID);
 ```
 
-This lets us know whether the *currently selected* friend is online. If we pick a different friend and update the `recipientID` state variable, our `useFriendStatus` Hook will unsubscribe from the previously selected friend, and subscribe to the status of the newly selected one.
+Esto nos permite saber cuando el amigo* actualmente seleccionado* esta en linea. Si elegimos un amigo diferente y actualizamos la variable de estado `recipientID`, nuestro Hook `useFriendStatus` va a desuscribirse de el amigo previamente seleccionado, y suscribirse al estado de el nuevo seleccionado.
 
-## `useYourImagination()`
+## `usaTuImaginación()`
 
-Custom Hooks offer the flexibility of sharing logic that wasn't possible in React components before. You can write custom Hooks that cover a wide range of use cases like form handling, animation, declarative subscriptions, timers, and probably many more we haven't considered. What's more, you can build Hooks that are just as easy to use as React's built-in features.
+Los Hooks personalizados ofrecen la flexibilidad de compartir la lógica que no era posible con los componentes de React antes. Puedes escribir Hooks personalizados que cubran una amplia gama de casos de uso como manejo de formularios, animación, suscripciones declarativas, temporizadores y probablemente muchos mas que no hemos considerado. Ademas, puedes construir Hooks que sean tan fáciles de usar como las características incorporadas de React.
 
-Try to resist adding abstraction too early. Now that function components can do more, it's likely that the average function component in your codebase will become longer. This is normal -- don't feel like you *have to* immediately split it into Hooks. But we also encourage you to start spotting cases where a custom Hook could hide complex logic behind a simple interface, or help untangle a messy component.
+Intenta resistirte a añadir la abstracción demasiado pronto. Ahora que los componentes funcionales pueden hacer mas, es probable que el promedio de componentes funcionales en tu base de código se amplíe. Esto es normal -- no te sientas como si *tuvieras* que dividirlo inmediatamente en Hooks. Pero también te animamos a empezar a descubrir casos donde un Hook personalizado podría ocultar la lógica compleja detrás de una interfaz simple, o ayudar a desenredar un componente desordenado.
```

Comment:
```suggestion
Intenta resistirte a añadir la abstracción demasiado pronto. Ahora que los componentes funcionales pueden hacer más, es probable que el promedio de componentes funcionales en tu base de código se amplíe. Esto es normal, no te sientas como si *tuvieras* que dividirlo inmediatamente en Hooks. Pero también te animamos a empezar a descubrir casos donde un Hook personalizado podría ocultar la lógica compleja detrás de una interfaz simple, o ayudar a desenredar un componente desordenado.
```

Reply:
I'd remove the article `la` in `Intenta resistirte a añadir la abstracción`, I believe they refer to abstraction in general not an abstraction in particular.